apps/api/src/app/shared/shared.module.ts

```diff
@@ -138,7 +139,14 @@ const PROVIDERS = [
 ];
 
 @Module({
-  imports: [],
+  imports: [
+    LoggerModule.forRoot(
+      createNestLoggingModuleOptions({
+        serviceName: 'novu/api',
+        version: '0.12.0',
```

Comment:
@Cliftonz why are we initializing it here twice? Once in the app module and the shared module? Should be only once, and if it's here let's make sure not to hard code those values